src/hooks/useActiveWeb3React.ts

```diff
@@ -0,0 +1,22 @@
+import { Web3Provider } from '@ethersproject/providers'
+import { useWeb3React } from '@web3-react/core'
+import { default as useWidgetsWeb3React } from 'lib/hooks/useActiveWeb3React'
+
+import { NetworkContextName } from '../constants/misc'
+
+export default function useActiveWeb3React() {
+  const widgetsContext = useWidgetsWeb3React()
+  const interfaceContext = useWeb3React<Web3Provider>()
+  const interfaceNetworkContext = useWeb3React<Web3Provider>(
+    process.env.REACT_APP_IS_WIDGET ? undefined : NetworkContextName
```

Comment:
Is this branching strictly necessary? Or could `NetworkContextName` always be passed here?

Reply:
It would work to do the branching at build time, right? just export the ternary?

Reply:
nvm that won't work.

I'm hesitant to fiddle more w/ the functionality here. Can follow up on this later.